src/NuGet.Core/NuGet.ProjectModel/PackageSpecUtility.cs

```diff
@@ -101,9 +112,16 @@ public static bool IsSnapshotVersion(string version)
                 throw new ArgumentNullException(nameof(projectFramework));
             }
 
+            var hasImports = imports?.Any() == true;
             var hasATF = assetTargetFallback?.Any() == true;
```

Comment:
can we be more verbose please? use hasAssetsTargetFallback and hasPackageTargetsFallback .